tests/scancode/data/help/help.txt

```diff
@@ -0,0 +1,76 @@
+Usage: scancode [OPTIONS] <input> <output_file>
+
+  scan the <input> file or directory for origin clues and license and save
+  results to the <output_file>.
+
+  The scan results are printed to stdout if <output_file> is not provided.
+  Error and progress is printed to stderr.
+
+Options:
+
+  scans:
```

Comment:
This is how `pytest -h` looks like, but I can exclude the `group` labels and just group options using extra indentation, if that's required.